frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunTabDetails.tsx

```diff
@@ -62,10 +62,12 @@ const PipelineRunTabDetails: React.FC<PipelineRunTabDetailsProps> = ({
     { key: 'Workflow name', value: workflowName },
     { key: 'Created at', value: asTimestamp(new Date(pipelineRunKF.created_at)) },
     {
-      key: 'Started at',
-      value: asTimestamp(new Date(pipelineRunKF.scheduled_at || pipelineRunKF.created_at)),
+      key: 'Finished at',
+      value:
+        pipelineRunKF.finished_at === '1970-01-01T00:00:00Z'
```

Comment:
Define a new constant and use it here: `const epoch = new Date(0).toISOString();`

```suggestion
        pipelineRunKF.finished_at === epoch
```